tests/fast-integration/skr-svcat-migration-test/deploy-sample-resources.js

```diff
@@ -175,10 +175,102 @@ function sleep(ms) {
 
 async function goodNight() {
   console.log('Taking a break...');
-  await sleep(900000); //15min
+  await sleep(1200000); //20min
   console.log('Waking up...');
 }
 
+async function storeSecretsAndPresets() {
+  let secretsResp = await getSecrets("default")
+  let podPresetsResp = await getPodPresets("default")
+
+  let secrets = []
+  let podPresets = []
+
+  secretsResp.forEach(function (secret) {
+    secrets.push(secret.metadata.name)
+  })
+
+  podPresetsResp.forEach(function (podPreset) {
+    podPresets.push(podPreset.metadata.name)
+  })
+
+  // return { secrets, podPresets }
+  return Promise
```

Comment:
What is the purpose of this Promise? As I understand this correctly you are just returning (resolving) the promise with JSON object having two properties (secrets and podPresets). This can be handled with just a return statement like:
```
return {secrets, podPresets}
```